### ISSUE TITLE
How do I convert the value of a HC<_> to an isize?

### ISSUE THREAD

**Issue by llogiq** [epsilonz/shoggoth.rs#37]
Hi there,

I'm currently looking into shoggoth.rs' nat implementation. One feature that appears to be missing is the ability to create an integer value from the type. Have I missed something or is it simply not implemented?

**Comment by freebroccolo** [comment]
Yeah, it's just not in the current version IIRC. I did have that implemented in some earlier attempts but I started to redo the type level nats to be much more efficient (based on [this](https://github.com/freebroccolo/hs-ackermann)). However, I put it on hold until the type macros RFC is resolved.

You should be able to implement conversion to isize by recursion in a similar fashion to the other operations though.

**Comment by llogiq** [comment]
Cool (at least once the build works again – seems to be some breakage related to macros).
